content/en/apps/features/supervision/supervision.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Supervision
+weight: 7
+description: >
+  Supervision and workforce management to strengthen health systems
+keywords: supervisor
+relatedContent: >
+  apps/features/targets
+  design/best-practices/#targets
+  apps/features/integrations/dhis2
+---
+
+Supervision and workforce management are important aspects to building and maintaining high performing community health systems. Supervisors help Community Health Workers to deliver quality healthcare services to their patients through: 1) building CHW care delivery knowledge and skills; 2) fostering a supportive work environment; and 3) supporting continuity of care between home-based care and community health centers or facilities. 
+
+The WHO’s [guidelines](https://www.who.int/hrh/community/guideline-health-support-optimize-hw-programmes/en/) on health policy and system support to optimize community-based health worker programmes emphasizes the importance of regular and systematic CHW supervision. Following these guidelines, the CHT is designed to enable Supervisors to provide personalized performance feedback during CHW supervision as well as track aggregate statistics. Data collected from completed [Care Guides]({{< relref "apps/concepts/care-guides" >}}) produce granular information that can be used to provide coaching to promote compliance with health program standards of practice and monitor outcomes.
+
+## CHW Aggregate Targets
+
+{{< figure src="aggregate-supervisor.png" link="aggregate-supervisor.png" class="right col-7 col-lg-6" >}}
+
+For CHW Supervisors, the [Targets]({{< relref "apps/features/targets" >}}) tab provides important insights into how their community unit is progressing towards their goals. It presents Supervisors with actionable information about their CHWs, by aggregating data for each of the CHWs that a Supervisor manages and presenting it in an easily digestible format. This enables Supervisors to gain insight into how well their team of CHWs is working together to meet common goals.
```

Comment:
Targets can give a more general view, and not explicitly about goals. Removed it from the first sentence, as we still get into how goals can be used further in the paragraph.

```suggestion
For CHW Supervisors, the [Targets]({{< relref "apps/features/targets" >}}) tab provides important insights into their community unit. It presents Supervisors with actionable information about their CHWs, by aggregating data for each of the CHWs that a Supervisor manages and presenting it in an easily digestible format. This enables Supervisors to gain insight into how well their team of CHWs is working together to meet common goals.
```